tools/RAiDER/statsPlot.py

```diff
@@ -807,11 +827,13 @@ def stats_analyses(inps=None):
 
     # Station plots
     # Plot each individual station
+    start_time = time.time()
     if inps.station_distribution:
         print("- Plot spatial distribution of stations.")
         unique_points = df_stats.df.groupby(['Lon', 'Lat']).size()
         df_stats([unique_points.index.get_level_values('Lon').tolist(), unique_points.index.get_level_values('Lat').tolist(
         )], 'station_distribution', workdir=os.path.join(inps.workdir, 'figures'), plotFormat=inps.plot_fmt)
+    print("station_distribution took", time.time() - start_time)
```

Comment:
I would put time statements under the verbose option, so they are only run if the user requests verbose output.

Reply:
As we discussed in the meeting, I will leave these out for now. What I think would be more useful is a progress bar. Will brainstorm in a separate issue ticket.